testnets/sgetestnet4/assetlist.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "sgetestnet4",
+  "assets": [
+    {
+      "description": "The SGE token is primarily a governance token for the SGE Network blockchain.",
+      "denom_units": [
+        {
+          "denom": "usge",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "usge",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "usge",
+      "name": "SGE Testnet4",
+      "display": "sge",
+      "symbol": "SGE",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/sgetestnet4/images/sge.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/sgetestnet4/images/sge.png"
```

Comment:
can we point these to the existing sge token logos instead? We have several sets of sge images when we only need one.

We could even use an image_sync and point it to the same image mainnet uses